### PR TITLE
[4667] Add doc for importing the monthly funding data

### PR DESCRIPTION
**Pull request by AbigailMcP**
### Context

So that it's not just in my notes app :)

### Changes proposed in this pull request

Add documentation for importing the monthly funding data into Register from provided csvs.

### Guidance to review

Check that it makes sense - could you do this next month?

### Important business

- [ ] ~Does this PR introduce any PII fields that need to be overwritten or deleted in db/scripts/sanitise.sql?~
- [ ] ~Does this PR change the database schema? If so, have you updated the config/analytics.yml file and considered whether you need to send 'import_entity' events?~

NB: Please notify the #twd_data_insights team and ask for a review if new fields are being added to analytics.yml

### CODE REVIEW

**Comment by AbigailMcP** [comment]
[Write docs for adding funding CSV](https://trello.com/c/sI9mKQyW/4667-write-docs-for-adding-funding-csv)

File: docs/uploading_funding_csvs.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    ```
+
+3. Check that you've definitely copied the correct thing! Paste it somewhere
+
+4. SSH onto register-worker-production, navigate into the app directory and
+  create a new file
+
+    ```bash
+    cf ssh register-worker-production
+    cd /app
```
Comment (markhallen) [inline docs/uploading_funding_csvs.md:53]: Even though we delete the file a bit later `/app/tmp` would probably be a better place for it.

**Review by markhallen — APPROVED** [review]
Spongebob Thumb

File: docs/uploading_funding_csvs.md (hunk 1 above)
Comment (AbigailMcP) [reply docs/uploading_funding_csvs.md]: Updated!

**Pull request merged**